rclpy/rclpy/impl/__init__.py

```diff
@@ -26,6 +26,18 @@ def _import(name):
         with add_dll_directories_from_env('PATH'):
             return importlib.import_module(name, package='rclpy')
     except ImportError as e:
+        if e.path is None:
+            import sysconfig
+            expected_path = os.path.join(
+                os.path.dirname(os.path.dirname(__file__)),
```

Comment:
nit: Prefer using `pathlib` over `os.path`

Reply:
Done in 8a1623b6be331821bfc461a54702bd0a12e179e1.